authors/apps/authentication/__init__.py

```diff
@@ -0,0 +1,15 @@
+from django.apps import AppConfig
+
+
+class AuthenticationAppConfig(AppConfig):
+    name = 'authors.apps.authentication'
+    label = 'authentication'
+    verbose_name = 'Authentication'
+
+    def ready(self):
+        import authors.apps.authentication.signals
+
+# This is how we register our custom app config with Django. Django is smart
```

Comment:
When I read these comments, I get the impression that you've copied and pasted this code.
Please ensure that when you're using code from a tutorial, you understand what it does and write it accordingly.
In this case that means you write (in layman's terms) what the introduction of this code into this `__init__.py` file does.

I also see missing new lines in this file and one or two others.
This would affect you negatively if you were working on a real product because it is not quality code.
Kindly make the necessary changes.

Reply:
The code in there helps for automation of Django to automatically pick up a signal every time a new user is created and hence use it to create an associated profile. Let me make more research on the would be missing lines please.

Reply:
There is no need to research about the missing new_lines. You simply have to press enter at the end of this file to get rid of this red mark 👇 
<img width="1129" alt="Screenshot 2019-04-10 at 17 54 06" src="https://user-images.githubusercontent.com/40039818/55889498-d14be480-5bb9-11e9-9317-7401971e43b5.png">

Regarding the comments, please replace it with what you've said above. Reading it, a person can tell that you are the one that wrote the comments, as opposed to what is currently there, from which I garnered that you'd gotten this section of code from somewhere else.

Reply:
I have resolved this please check back